src/main/java/duke/Event.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+/**
+ * Event encapsulates task that starts at a specific date / time and
+ * ends at a specific date/time
+ */
+public class Event extends Task {
+    private String taskName;
+    private String startTime;
+    private String endTime;
+    private DateTime start;
+    private DateTime end;
+
+    /**
+     * Constructor for creating an Event
+     *
+     * @param taskName name of task.
+     */
+    public Event(String taskName, DateTime start, DateTime end) {
+        super(taskName);
+        this.taskName = taskName;
+        this.startTime = start.getDateTime();
+        this.endTime = end.getDateTime();
+        this.start = start;
+        this.end = end;
+    }
+
+    /**
+     * Constructor for creating an Event based on whether its done or not
+     * @param taskName  name of task.
+     * @param isDone    whether the task is done or not
+     * @param start    the start time of the event
+     * @param end    the end time of the event
```

Comment:
```suggestion
     * @param taskName name of task.
     * @param isDone whether the task is done or not
     * @param start the start time of the event
     * @param end the end time of the event
```
Are these extra spaces supposed to be here?